java/ql/src/experimental/semmle/code/java/security/StaticInitializationVectorQuery.qll

```diff
@@ -28,7 +28,12 @@ private class StaticByteArrayCreation extends ArrayCreationExpr {
   }
 }
 
-/** Defines a sub-set of expressions that update an array. */
+/**
+ * Defines a sub-set of expressions that update either content of an array or an array reference.
+ * There sub-set covers only methods that are likely to set a non-static IV.
```

Comment:
Should probably be "Their":
```suggestion
 * Their sub-set covers only methods that are likely to set a non-static IV.
```

Reply:
Oops, I meant "this". Thanks for pointing out!